pallets/ocex/src/lib.rs

```diff
@@ -98,18 +129,118 @@ decl_module! {
         /// It helps to notify enclave about sender's intend to withdraw via on-chain
         #[weight = 10000]
         pub fn withdraw(origin, asset_id:  AssetId, to: T::AccountId,amount: T::Balance) -> DispatchResult{
-            let sender: T::AccountId = ensure_signed(origin)?;
+            let _: T::AccountId = ensure_signed(origin)?;
             Self::deposit_event(RawEvent::TokenWithdrawn(asset_id, to, amount));
             Ok(())
         }
 
+        /// Register MainAccount
+        #[weight = 10000]
+        pub fn register(origin) -> DispatchResult{
```

Comment:
what is the use case for tgis register function? Why not register proxies directly or deposit funds?

Reply:
Orderbook is only allowed for those who has done decentralized KYC using KILT protocol. We will add some extra logic here that do that check